### PR TITLE
Created ai-reviewer-admin app

### PR DESCRIPTION
**Pull request by KDNeufeld**
# Description

This PR includes the following proposed change(s):

https://justice.gov.bc.ca/jira/browse/FLA-1030

- created a basic react-app 
- set port of app to run on 3002
- added _ai-reviewer-admin_ to **docker-compose**
- added BC Gov header and footer
- added lint and got yarn test to work

image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

yarn lint
yarn test
yarn test --fix

## Does the change impact or break the Docker build?

- [x] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [x] Yes
- [ ] No

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/ai-reviewer-admin/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+Runs the app in the development mode.\
+Open [http://localhost:3002](http://localhost:3002) to view it in the browser.
+
+The page will reload if you make edits.\
+You will also see any lint errors in the console.
+
+### `yarn test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
```
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply src/frontend/ai-reviewer-admin/README.md:20]: generated readme

File: src/frontend/ai-reviewer-admin/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+### `yarn build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
```
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply src/frontend/ai-reviewer-admin/README.md:25]: generated readme

File: src/frontend/ai-reviewer-admin/README.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
```
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply src/frontend/ai-reviewer-admin/README.md:30]: generated readme

File: src/frontend/ai-reviewer-admin/README.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+### `yarn eject`
+
+**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
+
+If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
```
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply src/frontend/ai-reviewer-admin/README.md:36]: generated readme

File: src/frontend/ai-reviewer-admin/README.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
```
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply src/frontend/ai-reviewer-admin/README.md:38]: generated readme

File: src/frontend/ai-reviewer-admin/README.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
```
[the remark this answers is not in the document]
Comment (KDNeufeld) [reply src/frontend/ai-reviewer-admin/README.md:40]: generated readme

**Review by yousiefc — APPROVED** [review]
Beautiful :100:

**Pull request merged**